dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBVectorStore.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Microsoft.SemanticKernel.Data;
+using MongoDB.Driver;
+
+namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB;
+
+/// <summary>
+/// Class for accessing the list of collections in a Azure CosmosDB MongoDB vector store.
+/// </summary>
+/// <remarks>
+/// This class can be used with collections of any schema type, but requires you to provide schema information when getting a collection.
+/// </remarks>
+public sealed class AzureCosmosDBMongoDBVectorStore : IVectorStore
```

Comment:
Should we add unit tests for this and the collection and record classes?

Reply:
Definitely, as mentioned in description, I want to add unit tests in next follow-up PR to make review process easier, but I can also do it in scope of this PR in case we think it's critically to merge it as one bundle.

Reply:
It's not crucial to merge as one bundle, feel free to separate it.

Reply:
Added unit tests as part of this PR.